pkg/http/core/kubernetes/ops.go

```diff
@@ -77,6 +77,7 @@ type PatchManifestRequestOptions struct {
 	Record        bool   `json:"record"`
 }
 
+// why are artifacts commented out here? possibly causing the problem of artifacts not getting bound correctly
```

Comment:
If there's fields that I didn't *think* were needed in a request struct I would comment them out. It makes it easier to just uncomment them in the future when they are needed.

Always feel free to uncomment fields that are needed.